### PR TITLE
multi target netcoreapp 2.1 and 3.1

### PR DESCRIPTION
**Pull request by EamonHetherton**

### CODE REVIEW

**Comment by EamonHetherton** [comment]
Any chance of a new nuget package release with netcoreapp3.1 support?

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs [hunk 1]
```diff
@@ -8,269 +8,269 @@
...
-    public class AuthenticodeKeyVaultSigner : IDisposable
-    {
-        private readonly AsymmetricAlgorithm _signingAlgorithm;
-        private readonly X509Certificate2 _signingCertificate;
-        private readonly HashAlgorithmName _fileDigestAlgorithm;
-        private readonly TimeStampConfiguration _timeStampConfiguration;
-        private readonly MemoryCertificateStore _certificateStore;
-        private readonly X509Chain _chain;
-        private readonly SignCallback _signCallback;
+	/// <summary>
```
Comment (vcsjones) [inline src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs:11]: This converted the file to use tab indents. Can you please undo that?

**Comment by EamonHetherton** [comment]
changes waiting review...

File: src/AzureSignTool/AzureSignTool.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (vcsjones) [inline src/AzureSignTool/AzureSignTool.csproj:5]: I don’t think multi targeting works for global tools and “dotnet pack” will fail. I’m fine dropping netcoreapp2.1 entirely.

**Review by vcsjones — CHANGES_REQUESTED** [review]
Super sorry for the slow response on this. One small comment and I think we are good.

File: src/AzureSignTool/AzureSignTool.csproj (hunk 2 above)
Comment (vcsjones) [reply src/AzureSignTool/AzureSignTool.csproj:5]: https://github.com/dotnet/sdk/issues/9917 For some detail.

Comment (EamonHetherton) [reply src/AzureSignTool/AzureSignTool.csproj:5]: I’m fine dropping netcoreapp2.1 entirely too 👍

**Review by vcsjones — APPROVED** [review]

**Pull request merged**